app/components/dashboard/all_collections_row_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<tr>
+  <td><%= link_to collection.head.name, collection %><strong><i><%= draft_label %></i></strong></td>
```

Comment:
Should there be a space before the draft_label?

Reply:
Do we want to have `<strong><i></i></strong>` on the page even when there is no draft label?